procurement/templates/procurement/home.html

```diff
@@ -9,14 +9,16 @@
 <div class="bg-gray-100 border-top py-4 py-md-5">
     <div class="container" style="max-width: 44em">
 
-        <form class="row" action="" method="GET">
-            <fieldset class="col-sm-auto me-auto">
-                <label for="council-search" class="form-label">Search by postcode or council name</label>
-                <div class="search-input">
-                    <!-- { % include icons/search.html %} -->
-                    <input type="search" name="pc" id="council-search" class="form-control" value="{{ filter.form.pc.value }}">
-                </div>
-            </fieldset>
+        <form action="" method="GET">
+            <div class="row">
+                <fieldset class="col-sm-auto me-auto">
+                    <label for="council-search" class="form-label">Search by postcode or council name</label>
+                    <div class="search-input">
+                        {% include 'procurement/icons/search.html' with classes='mr-2 flex-grow-0 flex-shrink-0' width='1.25em' height='1.25em' role='presentation' %}
+                        <input type="search" name="pc" id="council-search" class="form-control" value="{{ filter.form.pc.value }}">
```

Comment:
`{{ filter.form.pc.value }}` here currently outputs `None` when a search hasn’t been performed yet. I guess we should probably use `{{ filter.form.pc.value|default_if_none:"" }}` to make sure the `value` attribute is empty when a search hasn’t been performed?

Reply:
One for @lucascumsille